src/assets/scripts/client/constants/inputConstants.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Context of commands entered into command bar
+ *
+ * @enum COMMAND_CONTEXT
+ * @type {object}
+ */
+export const COMMAND_CONTEXT = {
+    AIRCRAFT: 'aircraft',
+    SCOPE: 'scope'
+};
+
+/**
+* Enumeration of key codes used for inputs.
+*
+* @property KEY_CODES
+* @type {Object}
+* @final
+*/
+export const KEY_CODES = {
```

Comment:
almost threw up a little when I saw these in red up above. I should know better, this definitely belongs here 👍